src/server/string_family.h

```diff
@@ -44,6 +44,10 @@ class SetCmd {
 
   OpStatus Set(const SetParams& params, std::string_view key, std::string_view value);
 
+  constexpr const OpArgs& args() const {
```

Comment:
lets call it `op_args()` because when reading the code I first thought it passes the command line arguments